src/backend/demo/tests/test_commands_create_demo.py

```diff
@@ -1,29 +1,19 @@
 """Test the `create_demo` management command"""
 
-from unittest import mock
-
 from django.core.management import call_command
 from django.test import override_settings
 
 import pytest
 
 from core import models
 
-from demo import defaults
+from demo.defaults import NB_OBJECTS
 from mailbox_manager import models as mailbox_models
 
-TEST_NB_OBJECTS = {
-    "users": 5,
-    "teams": 3,
-    "max_users_per_team": 5,
-    "domains": 2,
-}
-
 pytestmark = pytest.mark.django_db
 
 
 @override_settings(DEBUG=True)
-@mock.patch.dict(defaults.NB_OBJECTS, TEST_NB_OBJECTS)
```

Comment:
why not mock? It allows to run it with less objects in tests?

Reply:
Yes, but the tests were not failing with the amount of objects in this tests, yet the command was failing. So I guess it was better to test the command as it is.